templates/register.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'base.html' %}
+
+{% block title %}Регистрация - Мой Интернет-магазин{% endblock %}
+
+{% block content %}
+    <h2>Регистрация</h2>
+    <form action="{{ url_for('register') }}" method="post">
+        {{ form.hidden_tag() }}
+        <label for="first_name">Имя:</label>
+        {{ form.first_name(class="form-control") }}
+        <br>
+        <label for="last_name">Фамилия:</label>
+        {{ form.last_name(class="form-control") }}
+        <br>
+        <label for="email">Email:</label>
+        {{ form.email(class="form-control") }}
+        <br>
+        <label for="password">Пароль:</label>
+        {{ form.password(class="form-control") }}
+        <br>
+        <label for="confirm_password">Подтвердить пароль:</label>
+        {{ form.confirm_password(class="form-control") }}
+        <br>
+        <input type="submit" value="Зарегистрироваться" class="btn btn-primary">
```

Comment:
У вас ведь есть кнопка в форме, зачем тогда ее использовать тут?
```suggestion
        {{ form.submit(class="btn btn-primary") }}>
```